packages/daemon/src/db/index.ts

```diff
@@ -776,7 +776,7 @@ export const getAddressWalletInfo = async (mysql: MysqlConnection, addresses: st
   }
 
   const addressWalletMap: StringMap<Wallet> = {};
-  const [results] = await mysql.query(
+  const [results, _]: [mysql.RowDataPacket[], mysql.FieldPacket[]] = await mysql.query(
```

Comment:
You can create a type for the rows and do:

```suggestion
  const [results, _]: [mysql.RowDataPacket[], mysql.FieldPacket[]] = await mysql.query<AddressWalletInfoRow>(
```

There are some examples in this file

Reply:
Done